packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -355,6 +355,10 @@ class AndroidWebViewController extends PlatformWebViewController {
     return _webView.settings.setMediaPlaybackRequiresUserGesture(require);
   }
 
+  /// Sets the text zoom of the page in percent. The default is 100.
```

Comment:
nit: Same here for `The default is 100.`. I missed this in the last pass.

Reply:
Thanks for being diligent. Fixed with: 5b3b2e81c34546bdebc5ee4e938d9cd8fb33480d